content/examples/02-next-pages/01-basics/01-generating-text.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Generating Text
+description: Learn to generate text using the Vercel AI SDK in your Next.js application
```

Comment:
wondering how we can describe the content more accurately (since it's used in search engines). Maybe "Example: Generating text with the Next.js pages router and the Vercel AI SDK"?